director/test/api/test_public.py

```diff
@@ -14,3 +14,5 @@ def test_get_user_failing_test(client):
     rv = client.get('/api/public/user/p3150133')
     assert rv.status_code == 404
 
+def add_user(client):
```

Comment:
Since this is not needed, maybe remove for now?
It doesn't seem to do anything anyway.

Reply:
true removing